ui/packages/catalog-ui-search/src/main/webapp/extension-points/table-export/table-export.tsx

```diff
@@ -165,19 +156,30 @@ export const getDownloadBody = (downloadInfo: DownloadInfo) => {
     getExportCount({ exportSize, selectionInterface, customExportCount }),
     properties.exportResultLimit
   )
-  const cql = getCqlForSize(exportSize, count, selectionInterface)
+  const cql = selectionInterface.getCurrentQuery().get('cql')
   const srcs = getSrcs(selectionInterface)
   const sorts = getSorts(selectionInterface)
+  const hiddenResults = getHiddenResults(exportSize)
   const args = {
     hiddenFields: hiddenFields.length > 0 ? hiddenFields : [],
     columnOrder: columnOrder.length > 0 ? columnOrder : {},
     columnAliasMap: properties.attributeAliases,
   }
+
+  const searches = srcs.map((src: string) => {
+    const start = getStartIndex(src, exportSize, selectionInterface)
+    return {
+      src,
+      cql,
+      start,
+    }
+  })
+
   return {
-    cql,
-    srcs,
+    searches,
     count,
     sorts,
+    hiddenResults,
```

Comment:
❓ what's the hidden results for?

Reply:
Results can be hidden on the ui and we don't want these to appear for an export `visible`. Previously `getActiveSearchResults` did this filtering by only retrieving ids of non-hidden results, now we are passing this list of ids to do this filtering on the backend